docs/android-core-new/participants/participant-object.mdx

```diff
@@ -4,116 +4,87 @@ description: The object corresponding to a particular participant.
-- `audioEnabled`: Set to true if the participant is unmuted.
-- `screenShareEnabled`: Set to true if the participant is sharing their screen.
-- `videoTrack`: The video track of the participant.
-- `audioTrack`: The audio track of the participant.
-- `screenShareTracks`: The video and audio (if any) track of the participant's
-  screen share stream.
-
-**Metadata**:
 - `id`: The `participantId` of the participant (aka `peerId`).
 - `userId`: The `userId` of the participant.
 - `name`: The participant's name.
 - `picture`: The participant's picture (if any).
 - `clientSpecificId`: An arbitrary ID that can be set to identify the
   participant.
-- `isPinned`: Set to true if the participant is pinned.
+- `videoTrack`: The video track of the participant.
+- `screenShareTrack`: The video and audio (if any) track of the participant's
+  screen share stream.
+- `videoEnabled`: Set to true if the participant's camera is on.
+- `audioEnabled`: Set to true if the participant is unmuted.
+- `isPinned`: True if current user is pinned in the meeting room.
 - `presetName`: Name of the preset associated with the participant.
+- `stageStatus`: Status of stage for the participant
 
-The participant object is an event emitter, so you can set listeners on this
-object for events such as video and audio updates. For instance, to fire a
-callback when a participant toggles their mic, you can subscribe to the
-following events.
-
-```ts
-meeting.participants.joined
-  .get(participantId)
-  .on('audioUpdate', ({ audioEnabled, audioTrack }) => {
-    // This will only be fired on mic toggles for the participant with ID `participantId`
-    console.log(
-      'The participant with id',
-      participantId,
-      'has toggled their mic to',
-      audioEnabled
-    );
-  });
-```
+## To get video view of a given participant
 
-The events emitted by all participant objects are also re-emitted by all the
-maps in `meeting.participants`. Therefore, you can add a listener to
-`meeting.participants.joined` for the `audioUpdate` event. For instance, the
-same code above can be re-implemented as follows.
-
-```ts
-meeting.participants.joined.on(
-  'audioUpdate',
-  (participant, { audioEnabled, audioTrack }) => {
-    // This will be fired on mic toggles for all participants in the meeting
-    console.log(
-      'The participant with id',
-      participant.id,
-      'has toggled their mic to',
-      audioEnabled
-    );
-  }
-);
-```
+You can call `participant.getVideoView()` which will return a View which further
+can used to add in any ViewGroup in android.
+
+Similarly one can use `participant.getScreenShareView()` which will return a
+View which further can used to add in any ViewGroup in android.
+
+## Host Controls
+
+If you are the host of the room, you will see the **host controls** when you click the **three-dot menu** on the top right corner of the participants list. The host controls allow you to manage the participants in the room.
```

Comment:
(change-request): This description suits more for the UI Kit docs. Can we use the description for the existing core docs?
android-core: https://docs.dyte.io/android-core/participants/participant-object#host-controls-methods